test/test_creep.py

```diff
@@ -225,6 +245,62 @@ def test_g(self):
     g_model = self.model.g(self.s, self.e, self.t, self.T)
     self.assertTrue(np.isclose(g_calc, g_model))
 
+class Test225CreepCase1(unittest.TestCase, CommonScalarCreep):
+  def setUp(self):
+    self.model = creep.MinCreep225Cr1MoCreep() 
+
+    self.T = 500 + 273.15
+    self.e = 0.1
+    self.s = 50.0
+    self.t = 10.0
+
+    self.U = interpolate.PiecewiseLinearInterpolate(
+        [371, 400, 450, 500, 550, 600, 621, 649],
+        [471,468,452,418,634,284,300,270])
+
+  def test_g(self):
+    g_direct = self.model.g(self.s, self.e, self.t, self.T)
+    Uv = self.U.value(self.T - 273.15)
+    g_calc = 10.0**(6.7475+0.011426*self.s+987.72/Uv * np.log10(self.s) - 13494.0 / self.T) / 100.0
+    self.assertTrue(np.isclose(g_direct, g_calc))
+
+class Test225CreepCase2(unittest.TestCase, CommonScalarCreep):
+  def setUp(self):
+    self.model = creep.MinCreep225Cr1MoCreep() 
+
+    self.T = 500 + 273.15
+    self.e = 0.1
+    self.s = 100.0
+    self.t = 10.0
+
+    self.U = interpolate.PiecewiseLinearInterpolate(
+        [371, 400, 450, 500, 550, 600, 621, 649],
+        [471,468,452,418,634,284,300,270])
+
+  def test_g(self):
+    g_direct = self.model.g(self.s, self.e, self.t, self.T)
+    Uv = self.U.value(self.T - 273.15)
+    g_calc = 10.0**(6.7475+0.011426*self.s+987.72/Uv * np.log10(self.s) - 13494.0 / self.T) / 100.0
+    self.assertTrue(np.isclose(g_direct, g_calc))
+
+class Test225CreepCase3(unittest.TestCase, CommonScalarCreep):
+  def setUp(self):
+    self.model = creep.MinCreep225Cr1MoCreep() 
+
+    self.T = 600 + 273.15
+    self.e = 0.1
+    self.s = 100.0
+    self.t = 10.0
+
+    self.U = interpolate.PiecewiseLinearInterpolate(
+        [371, 400, 450, 500, 550, 600, 621, 649],
+        [471,468,452,418,634,284,300,270])
+
+  def test_g(self):
+    g_direct = self.model.g(self.s, self.e, self.t, self.T)
+    Uv = self.U.value(self.T - 273.15)
+    g_calc = 10.0**(11.498-8.2226*Uv/self.T-20448/self.T+5862.4/self.T*np.log10(self.s))/100.0
```

Comment:
might help in clarity if there is a space between the individual terms

Reply:
Okay, done.